docs/user_guide.rst

```diff
@@ -156,16 +156,59 @@ Now we can study how the circuit gets transpiled:
                                                                                               ║
            meas_2: ═══════════════════════════════════════════════════════════════════════════╩═
 
-We can also simulate the execution of the transpiled circuit before actually executing it:
+
+Simulating the execution of a transpiled circuit locally
+--------------------------------------------------------
+We can also simulate the execution of any transpiled circuit locally before actually executing it on real hardware by running it like this:
 
 .. code-block:: python
 
-    from qiskit import Aer
+    from qiskit import execute, QuantumCircuit
+    from qiskit_iqm import IQMFakeAdonis
+    from qiskit.compiler import transpile
+
+    circuit = QuantumCircuit(2)
+    circuit.h(0)
+    circuit.cx(0, 1)
+    circuit.measure_all()
+
+    backend = IQMFakeAdonis()
+    circuit = transpile(circuit, backend=backend)
+    job = execute(circuit, backend, qubit_mapping=None, shots=1000)
+    job.result().get_counts()
+
+
+This script uses an :class:`.IQMFakeAdonis` instance to conduct a noisy simulation of a circuit that was transpiled specifically to an IQM device.
+
+To use a noisy simulation you can import a `IQMFakeAdonis`. We have provided a chip sample that is used as a standard when creating an instance of
+`IQMFakeAdonis`. If you want to use your own noise parameters, you can just create your own chip sample by adapting the following template:
```

Comment:
Chip sample isn't a very informative word choice for an end user. I'd suggest "QPU sample" or "QPU specimen" or something like that.

```suggestion
:class:`IQMFakeBackend` can be initialized using an :class:`IQMChipSample` instance, which describes the
physical properties of the QPU specimen to be simulated. If not provided, by default each :class:`IQMFakeBackend` subclass uses representative values for that QPU type.


`IQMFakeAdonis`. If you want to use your own noise parameters, you can just create your own chip sample by adapting the following template:
```